### PR TITLE
Feature add database

### PR DESCRIPTION
**Pull request by Mhamad-Raad**
# Database in this branch I have done the followings: 

- Created all the needed tables.
- Created foreign keys for the table relationships.
- Connected the database.

### Done 💯 🎉

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @Mhamad-Raad,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Your readme documentation is not descriptive of this project. Kindly update your readme sections to be descriptive of this project. It contains details about the Ruby capstone

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,241 @@
...
+
+I would like to thank:
+
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```
Comment (1GORDON) [inline README.md:237]: Great job @Mhamad-Raad 🤝

- [ ] [Optional] You may fix the broken MIT license link to make your readme file more professional.

File: db/migrate/20230117072559_create_like.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
```
Comment (1GORDON) [inline db/migrate/20230117072559_create_like.rb:3]: Great job @Mhamad-Raad 🤝

- [ ] [Optional] You may add `t.references :author, foreign_key: { to_table: 'users' }` to the likes, comments, and posts tables.

File: db/migrate/20230117072528_create_comment.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :AuthorId
+      t.integer :PostId
+      t.string :Text
```
Comment (1GORDON) [inline db/migrate/20230117072528_create_comment.rb]: Great job @Mhamad-Raad 🤝

- [ ] [Optional] You may also make sure the Text is of type text. `t.text :text`.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Mhamad-Raad ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**